packages/cli/src/commands/generate/scaffold/scaffold.js

```diff
@@ -359,6 +359,20 @@ const componentFiles = async (
   const intForeignKeys = intForeignKeysForModel(model)
   let fileList = {}
   const componentMetadata = {
+    Enum: {
+      componentName: 'RadioField',
+      defaultProp: 'defaultChecked',
+      validation: () => false,
+      listDisplayFunction: 'formatEnum',
+      displayFunction: 'formatEnum',
+    },
+    EnumList: {
+      componentName: 'CheckboxField',
+      defaultProp: 'defaultChecked',
+      validation: () => false,
```

Comment:
Should we also support `isRequired` here as we do for some other types?

Reply:
I had to remove required validation for the enum list since

`colors[0]` and `colors[1]` is for names didn't match the `colors` requiredness.

But maybe the single enum can still

Reply:
Here's an example of validating checkboxes with react-hook-form https://github.com/react-hook-form/react-hook-form/issues/377#issuecomment-1026757806

Reply:
Note sure I know how to apply this. Could we do this PR and then refactor and optimize later?

Reply:
Sure. Let's do that!